pkgs/tools/text/wrap/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, buildGoModule, fetchFromGitHub, fetchpatch, makeWrapper, courier-prime }:
+
+buildGoModule rec {
+  pname = "wrap";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "Wraparound";
+    repo = "wrap";
+    rev = "v${version}";
+    sha256 = "0scf7v83p40r9k7k5v41rwiy9yyanfv3jm6jxs9bspxpywgjrk77";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  vendorSha256 = "03q5a5lm8zj1523gxkbc0y6a3mjj1z2h7nrr2qcz8nlghvp4cfaz";
+
+  patches = [
+    (fetchpatch {
+      name = "courier-prime-variants.patch";
+      url = "https://github.com/Wraparound/wrap/compare/v0.3.1...b72c280b6eddba9ec7b3507c1f143eb28a85c9c1.patch";
+      sha256 = "0cw6vsc37j748kzsyr2w8cx1pmm19z1pij95pk25204yqljbbz84";
+    })
+  ];
+
+  postInstall = ''
+    wrapProgram $out/bin/wrap --prefix XDG_DATA_DIRS : ${courier-prime}/share/
+  '';
+
+  meta = with lib; {
+    description = "A Fountain export tool with some extras";
+    homepage = "https://github.com/Wraparound/wrap";
+    license = licenses.gpl3;
```

Comment:
gpl3Only or gpl3Plus?

Reply:
[Project says GPL 3.0](https://github.com/Wraparound/wrap/blob/develop/LICENSE) so I think it's right, or is there some distinction I am missing?

Reply:
https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md

Reply:
Thanks, that was helpful! I think I have it right now.